server/src/app.ts

```diff
@@ -13,16 +13,17 @@ dbConnect();
 app.use(express.json({ limit: "10mb" }));
 app.use(cors());
 
-if (process.env.NODE_ENV !== "test") {
-  app.use(morgan("combined"));
-}
+app.use(morgan("dev"));
```

Comment:
Can we disable morgan logging for the test env? The console logs with morgan are not very nice to see :)